src/axom/mint/docs/sphinx/sections/getting_started.rst

```diff
@@ -99,16 +159,19 @@ supplying the third argument in the ``createField()`` method invocation.
     that the number of components of the field defaults to :math:`1` and thereby
```

Comment:
Should the `2` string literal be replaced by a variable like `dim` to make it more obvious what this parameter is for?

Reply:
Done.